src/main/java/com/cloudogu/smeagol/wiki/infrastructure/ScmGitCommitRepository.java

```diff
@@ -0,0 +1,43 @@
+package com.cloudogu.smeagol.wiki.infrastructure;
+
+import com.cloudogu.smeagol.wiki.domain.*;
+import com.google.common.base.Throwables;
+import org.eclipse.jgit.api.errors.GitAPIException;
+import org.eclipse.jgit.lib.PersonIdent;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.eclipse.jgit.lib.PersonIdent'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)